src/screens/SendFunds/AmountInput.js

```diff
@@ -166,24 +168,25 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   wrapper: {
-    minHeight: 100,
+    flexBasis: 100,
+    flexShrink: 0.5,
     flexDirection: "column",
     justifyContent: "center",
   },
   currency: {
-    fontSize: 24,
+    fontSize: normalizeZize(20),
```

Comment:
I'm not sure of variable font size idea. we don't do that anywhere else and that would feel weird if part of the app start doing that. I would like to hear from design too.

to me, fixing by flex should be enough, it's not?

Reply:
Just fixed the issue without the need to normalize text font size. I figured that this particular active font size was rather big but it can fit in the 5S screen just right if the other elements don't grow out of flex space :+1: